src/main/java/com/dev/cinema/exceptions/AuthenticationException.java

```diff
@@ -0,0 +1,7 @@
+package com.dev.cinema.exceptions;
+
+public class AuthenticationException extends RuntimeException {
```

Comment:
```suggestion
public class AuthenticationException extends Exception {
```
it's not critical when you have entered the wrong password)
You cannot avoid it programmatically.

Reply:
done